src/04-arrays.js

```diff
@@ -135,7 +163,11 @@ function removeElementFromFrontOfArray(array) {}
  *  getMiddleElement([ 10, null, "30" ]);
  *  //> null
  */
-function getMiddleElement(array) {}
+function getMiddleElement(array) {
+let mid = array[Math.floor((array.length - 1) / 2)];
```

Comment:
Nice check with the Math.floor but considering the arrays are always odd, do we need it?